rust-semantics/rust-common-syntax.md

```diff
@@ -195,7 +195,12 @@ https://doc.rust-lang.org/reference/items/extern-crates.html
 
 ```k
 
-  syntax Struct ::= "TODO: not needed yet, not implementing"
+  syntax Structs ::= List{Struct, ""}
+  syntax Struct ::= StructStruct | TupleStruct
+  syntax TupleStruct ::= "TODO: not needed yet, not implementing"
+  syntax StructStruct ::= "struct" Identifier MaybeGenericParams "{" StructFields "}"
```

Comment:
Could you either match the reference, or add some TODOs where the implementation is different? Here you're missing `WhereClause?`, which is fine since we don't need it. Something similar applies to "StructField" below.